rpc/src/client/subscription.rs

```diff
@@ -411,6 +421,7 @@ impl SubscriptionRouter {
 
     /// Obtain a mutable reference to the subscription with the given ID (if it
     /// exists).
+    #[cfg(feature = "websocket-client")]
```

Comment:
Do we need these on every method or can we just put them on the struct def?

Reply:
See comment above